store/tools/ics23-iavl/.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+# This is a bit complex with multiple jobs.
+# https://circleci.com/docs/2.0/local-cli/ will let you test this locally
+version: 2
```

Comment:
can this file be deleted please. We can add GitHub Actions or it may be covered already

Reply:
Ah yes, meant to delete these. I think the existing test actions will cover it.